Source/WebKit/WebProcess/Plugins/PDF/PDFPluginTextAnnotation.mm

```diff
@@ -81,9 +83,7 @@ static const String cssAlignmentValueForNSTextAlignment(NSTextAlignment alignmen
 Ref<Element> PDFPluginTextAnnotation::createAnnotationElement()
 {
     Document& document = parent()->document();
-ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-    PDFAnnotationTextWidget *textAnnotation = this->textAnnotation();
-ALLOW_DEPRECATED_DECLARATIONS_END
+    PDFAnnotation *textAnnotation = annotation();
```

Comment:
`RetainPtr`

Reply:
Fixed in [116c479](https://github.com/WebKit/WebKit/commit/116c47933b9948f1c2e49242b9a652d27f3dee49)